include/mc_control/fsm/states/Meta.h

```diff
@@ -25,11 +25,8 @@ namespace fsm
  * - configs: can contain additional configuration for the states in the FSM
  *
  */
-
 struct MC_CONTROL_FSM_STATE_DLLAPI MetaState : State
 {
-  void configure(const mc_rtc::Configuration & config) override;
-
```

Comment:
`State::configure()` is purposefully not public because `State::configure_()` (resp. `State::start_` and `State::teardown_`) is the public interface that calls `State::configure()` so this is the function that should be called by `MotionFSM` in the linked project as it enables the common behaviors in all states (contacts/collisions manipulation, tasks/constraints addition)

The trailing `_` is a bit misleading but the common options came into existence after the FSM feature had been in the wild for a while

Reply:
Thanks for the comments, I understand.

So, is the following PR OK with your quick glance (sorry to delegate the review)?
https://gite.lirmm.fr/jrp-2019/mc_hwm_planner/-/merge_requests/38